awscli/Dockerfile

```diff
@@ -1,13 +1,11 @@
 FROM python:3.6-alpine3.7 as resource
 RUN apk add --update \
-        git \
-        zip \
-        unzip \
-        jq \
-        && rm -rf /var/cache/apk/*
+    git \
+    zip \
+    unzip \
+    jq \
+    && rm -rf /var/cache/apk/*
```

Comment:
As of Alpine Linux 3.3 there exists a new --no-cache option for apk. It allows users to install packages with an index that is updated and used on-the-fly and not cached locally.